### ISSUE TITLE
add external extensions

### ISSUE THREAD

**Issue by Nik1990** [Autodesk-Forge/viewer-react-express-headless#23]
Hi, if i would add an external extension such as this:

https://forge-rcdb.autodesk.io/configurator?id=5a361b2b58144b5ed2936ddf

here the source:

https://github.com/Autodesk-Forge/forge-rcdb.nodejs/tree/master/src/client/viewer.components/Viewer.Extensions.Dynamic/Viewing.Extension.PointCloudMarkup 

What i do? I have difficult with react code and don't understand how can add (and if is possible) external extensions

Thanks!

**Comment by jaimerosales** [comment]
Have you see our Tutorial material, might help you clarify future questions as well. http://learnforge.autodesk.io/#/ This does not use React. There is a section dedicated to basic skeletons of Extensions. I would start from there. Then you can move forward to use the RCDB Extensions which can be a bit more complicated, due to (React, Redux, ES6, WebPack) and so on.

**Comment by Nik1990** [comment]
Yes, i followed that tutorial and completed. Next i added the simple extension in that tutorial in current react based project. My next step is uderstend if is possible add other extension. I can't find any tutorial/docs that explain use of existing react extension. The RCDB Rxtensions repo seems good by less documented.

Any suggest?

Thanks
